lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3FileSystemConfig.java

```diff
@@ -224,6 +242,32 @@ public S3FileSystemConfig setCannedAcl(ObjectCannedAcl objectCannedAcl)
         return this;
     }
 
+    public RetryMode getRetryMode()
+    {
+        return retryMode;
+    }
+
+    @Config("s3.retry-mode")
+    @ConfigDescription("Specifies how the AWS SDK attempts retries, default is LEGACY")
+    public S3FileSystemConfig setRetryMode(RetryMode retryMode)
+    {
+        this.retryMode = retryMode;
+        return this;
+    }
+
+    @Min(1) // minimum set to 1 as the SDK validates this has to be > 0
+    public int getMaxErrorRetries()
+    {
+        return maxErrorRetries;
+    }
+
+    @Config("s3.max-error-retries")
+    public S3FileSystemConfig setMaxErrorRetries(int maxErrorRetries)
```

Comment:
Add `@ConfigDescription`

Reply:
I just left it as the config name is self-explanatory